src/ign-frontend/src/views/Home.vue

```diff
@@ -76,7 +76,7 @@
               <div class="circle"></div>
           </div>
           <div>
-            <strong>Total converted images: </strong>{{ apiCount }}
+            <strong>Total converted images: </strong>{{ apiCount.toString().replace(/[^\d]|[\']/g, '') }}
```

Comment:
```suggestion
            <strong>Total converted images: </strong>{{ apiCount.toString().replace(/^b'|'$/g, '') }}
```

even this solution have the same readability. Drop it if you dont like it.